inductiva/tests/mixins/test_file_manager.py

```diff
@@ -0,0 +1,201 @@
+"""Tests for the file manager mixin."""
+import os
+import pathlib
+
+import pytest
+
+from inductiva import mixins
+
+TEST_DIR = os.path.join(os.path.dirname(__file__), "assets")
+
+
+@pytest.mark.parametrize("name, listnames, expected_output",
+                         [("test", ["inductiva", "."], "test"),
+                          ("test", ["inductiva", "test"], "test#2"),
+                          ("test", ["inductiva", "test#2"], "test#3"),
+                          ("test", ["inductiva", "test-2"], "test")])
```

Comment:
imagine `gen_suffix` is hard coded to deal with the `"test"` string. It might be reasonable to have different test fixtures.